profiles/common.nix

```diff
@@ -8,6 +8,15 @@
     ./common/zfs.nix
   ];
 
+  # Use systemd during boot as well
+  boot.initrd.systemd.enable = true;
+
+  # Use EFI boot everywhere
+  boot.loader.efi.canTouchEfiVariables = true;
```

Comment:
This will fill up your EFI flash on every nixos-rebuild and potentially destroy cheap chips. It should be only done during installation.

Reply:
Good to know, thanks